src/param.rs

```diff
@@ -568,20 +568,28 @@ impl AudioProcessor for AudioParamProcessor {
         let period = 1. / scope.sample_rate.0 as f64;
         let param_intrisic_values = self.tick(scope.current_time, period, RENDER_QUANTUM_SIZE);
 
-        let input = &inputs[0]; // single input mode
         let param_computed_values = &mut outputs[0];
 
         param_computed_values
             .channel_data_mut(0)
             .copy_from_slice(param_intrisic_values);
 
-        param_computed_values.add(input, ChannelInterpretation::Discrete);
+        // add input signal to param computed values if not silent
+        let input = &inputs[0];
+
+        if !input.is_silent() {
```

Comment:
This optimization is already present in the `add` implementation of `AudioRenderQuantum` so we can keep this code clean

Reply:
This is a very tricky call to make. You might improve the current benchmark a bit, but we don't know the effect on all other programs. By checking for silence twice, you add extra work for all programs that handle non-silent inputs

Reply:
I would say that `AudioParam` are more often *not* modulated by an audio signal than the other way. But ok, let's keep that for another time with a more precise bench considering the two cases side by side, so we will have a clearer picture of the trade-off